.github/scripts/monday/removeLabel.js

```diff
@@ -22,22 +24,18 @@ module.exports = async ({ context }) => {
   }
 
   const tokensLabels = [designTokens, tokensPackage];
-  const remainingTokenLabel = tokensLabels.includes(labelName) && tokensLabels.find((l) => l !== labelName);
+  const remainingTokenLabel =
```

Comment:
I think you could simplify this to `const remainingTokenLabel = tokensLabels.find((l) => l !== labelName);` since it's checked for truthiness right after.

Reply:
Unfortunately I don't think so. If the script is run for any label other than one of those two `tokensLabels`, the `remainingTokenLabel` will return `designTokens` as it is the first label in the `tokensLabels` array that does not equal the current `labelName`. It is trying to check if the `labelName` **is** one of those two labels and if the other one is also attached to the issue.

Since this caught your eye, maybe the ternary idea we chatted about a bit ago would be more clear. WDYT?
```suggestion
  const remainingTokenLabel =
    labelName === designTokens
      ? tokensPackage
      : labelName === tokensPackage
        ? designTokens
        : null;
```

Reply:
That seems easier to read and understand. Nice!